.github/workflows/automatic-doc-checks.yml

```diff
@@ -0,0 +1,16 @@
+name: Main Documentation Checks
+
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true  
+
+jobs:
+  documentation-checks:
+    uses: canonical/documentation-workflows/.github/workflows/documentation-checks.yaml@main
+    with:
+      working-directory: doc/sphinx
```

Comment:
Couldn't this be `build_doc/doc/sphinx`, and then you wouldn't need to change the individual check targets?

Reply:
`build_doc/doc/sphinx` doesn't exist before the job is  run. For that to happen, we'd need to run a configure job first, upload `build_docs`,  then download it again _as a step of the documentation-checks job_. If we're going to edit it either way, might as well add a step to configure in `documentation-checks` itself.

Reply:
Right, because the install step's _in_ that `Makefile`… yeah, we need a step in between. Anyway, fine for now.